src/middlewared/middlewared/plugins/ftp.py

```diff
@@ -71,6 +72,7 @@ async def ftp_extend(self, data):
         Int('ipconnections', validators=[Range(min=0, max=1000)]),
         Int('loginattempt', validators=[Range(min=0, max=1000)]),
         Int('timeout', validators=[Range(min=0, max=10000)]),
+        Int('timeout_notransfer', validators=[Range(min=0, max=10000)]),
```

Comment:
Should we have `default=300` here too?

Reply:
What for?

Reply:
just a question as UI might want to skip this field in payload not get validation error 🤔  Not something that I am suggesting – we're good!

Reply:
@Qubad786 none of the fields are required for `update` methods, they work like HTTP PATCH.

Reply:
Ah makes sense.